firefox_puppeteer/tests/test_toolbars.py

```diff
@@ -135,3 +137,57 @@ def test_matching_text(self):
             self.assertTrue(len(all_matches) > 0)
             for match_fragment in all_matches:
                 self.assertIn(match_fragment, (input_text, input_text.upper()))
+
+
+class TestIdentityPopup(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+        self.identity_popup = self.browser.navbar.locationbar.identity_popup
+
+    def test_identity_popup_elements(self):
+        self.assertEqual('box', self.identity_popup.box.get_attribute('localName'))
```

Comment:
The expected value you are comparing with should be the second argument everywhere.